src/main/locales/en/bf_locale.json

```diff
@@ -0,0 +1,414 @@
+{
+    "LOCALE": {
+        "message": "en",
+        "description": "Max length:  2; Current language in short form: en, da, es, fr, nl ..."
+    },
+    "STR_LOCALE_SETUP": {
+        "message": "Language:",
+        "description": "Max length: 10; translation of: Language:"
+    },
+    "STR_COMMA": {
+        "message": ",",
+        "description": "Max length:  1; translation of: Comma string ,"
+    },
+    "STR_PERIOD": {
+        "message": ".",
+        "description": "Max length:  1; translation of: Period string ."
+    },
+    "STR_THOUSAND": {
+        "message": ".",
+        "description": "Max length:  1; translation of: Thousand delimiter ."
+    },
+    "STR_SECONDS": {
+        "message": "seconds",
+        "description": "Max length:  8; translation of: seconds"
+    },
+    "STR_VISUAL_BEEP": {
+        "message": "  * * * *",
+        "description": "Max length: 10; translation of:   * * * *"
+    },
+    "CMS_STARTUP_HELP_TEXT1": {
+        "message": "MENU:THR MID",
+        "description": "Max length: 20; translation of: MENU:THR MID"
+    },
```

Comment:
Some of the formats accepted by Crowdin, have an specific field for the max lenght. Using one of this formats, makes this max length mandatory in Crowdin. A comment maybe is not enough for the translators.

Reply:
I did look for the format for this max lenght, didn't found and made this solution.
Do you know the jason format ?

Reply:
Not for the JSON format, for this reason I suggested to use the Android XML format. It supports all the features we need.

Reply:
Right, and are supported by Crowdin ?

Reply:
It supports a lot of formats: https://store.crowdin.com/android-xml

Reply:
I look into the format, thx

Reply:
@McGiverGim , now I use the xml format, checkout bf_locale.xml
also auto generate bf_locale.h by python script, and here maxLength of translation are checked